hd/etc/welcome.txt

```diff
@@ -111,33 +113,46 @@ fr: Les <strong>droits des magiciens</strong> sont actuellement <strong>suspendu
       </div>
     </div>
   %end;
-  <div class="d-flex-inline flex-column text-lg-center mt-2">
-    %if;(bvar.motd != "")
+  %if;(bvar.motd != "")
+    <div class="d-flex flex-column flex-md-row justify-content-lg-center mt-2 mt-lg-4">
+      <div >
       <h3 class="font-weight-light">%apply;interp(bvar.motd)</h3>
-    %end;
-    %if;(browsing_with_sosa_ref and bvar.visitor_access!="no")<h4>%apply;display_sosa_ref%with;%sosa_ref;%end;</h4>%end;
-  </div>
-  %if;(bvar.visitor_access!="no")
+      %if;(browsing_with_sosa_ref and bvar.visitor_access!="no")<h4>%apply;display_sosa_ref%with;%sosa_ref;%end;</h4>%end;
+      </div>
+    </div>
+  %end;
+
+  %if;(bvar.visitor_access!="no" or wizard or friend)
     <div class="row justify-content-center mt-3 mt-md-4 mt-lg-5">
-      <div class="col-lg-7 ml-lg-4 ml-xl-5 h4"><i class="fa fa-search fa-sm fa-fw mr-1 text-secondary"></i>[*search/case sensitive]0</div>
+      <div class="col-lg-7 ml-lg-4 ml-xl-5 h4">
+        <i class="fa fa-search fa-sm fa-fw mr-1 text-secondary"></i>[*search/case sensitive]0</div>
```

Comment:
Not related to this PR.

Reply:
idem

Reply:
Sauf si mes yeux me jouent des tours, c'est un changement d'indentation... Donc bon, c'est pas super related quand même...

M'enfin bon, ça m'importe peu. C'est juste que ça aide pas à la relecture...